### PR TITLE
Upload images

### PR DESCRIPTION
**Pull request by runnersaw**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -26,7 +27,7 @@ var app = express();
 
 //Passport setup
 app.use(session({
-	secret: 'NotASecret;)',
+	secret: auth.secret,
```
Comment (cwallac) [inline app.js:30]: Life is now a secret

File: routes/routes.js [hunk 2]
```diff
@@ -7,7 +9,7 @@ module.exports = function(app, passport) {
 	});
 
 	app.get("/login", function(req, res) {
-		console.log("Get login")
+		console.log("Get login");
```
Comment (cwallac) [inline routes/routes.js:12]: Good thing you caught this issue

File: routes/routes.js [hunk 3]
```diff
@@ -30,6 +32,56 @@ module.exports = function(app, passport) {
 		res.sendFile(path.join(__dirname, '../views', 'profile.html'));
 	});
 
+	app.get("/uploadImage", function(req, res) {//isLoggedIn, function(req, res) {
```
Comment (cwallac) [inline routes/routes.js:35]: This isn't very RESTful...

File: routes/routes.js [hunk 4]
```diff
@@ -30,6 +32,56 @@ module.exports = function(app, passport) {
...
+		res.sendFile(path.join(__dirname, '../views', 'uploadImage.html'));
+	});
+
+	app.post("/uploadImage", function(req, res) {//isLoggedIn, function(req, res) {
+		var form = new multiparty.Form();
+
+		form.parse(req, function(err, fields, files) {
+
+			var img = files.image[0];
+			console.log(img);
```
Comment (cwallac) [inline routes/routes.js:45]: This is a quality log statement

File: routes/routes.js (hunk 3 above)
Comment (rvanderheyde) [inline routes/routes.js:35]: Are you talking about the naming there?

Comment (cwallac) [inline routes/routes.js:35]: I'm just posting random shit. None of this is serious just playing with sawyer

**Pull request merged**